set_tag.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Same thing here, better use POSIX and not rely on bash since the script does nothing that needs bash specifics.

```sh
#!/bin/sh -e

echo "TAG TO BE BUILT: '$INPUT_VERSION'
::set-output name=tag::${INPUT_VERSION:-latest}"
```

(Using a single `echo` call is also more efficient.)

Reply:
True

We can probably get rid of the first echo call

It was a dev echo for testing

Would you like to add a commit to update?

Reply:
Updating now